lib/src/anchor.dart

```diff
@@ -86,9 +86,11 @@ class Anchor {
       return _valueNames.entries.singleWhere((e) => e.value == name).key;
     } else {
       final regexp = RegExp(r'^\Anchor\(([^,]+), ([^\)]+)\)');
-      final matches = regexp.allMatches(name).first.group;
```

Comment:
matches is never null here

Reply:
Does it throw an exception if there is no match when calling first?

Reply:
let me add a test for that case